src/main/scala/Main.scala

```diff
@@ -1,8 +1,84 @@
+
+
+import newmodel.Package
+import newmodel._
+
+import scala.collection.mutable.ListBuffer
+
 /**
  * Created by yaroslav on 11.05.2015.
  */
 object Main {
-  def main(args:Array[String]) ={
-    println("hello world")
+
+  object DummyIndex extends Index{
+    val docElements = ListBuffer.empty[DocElement]
+    override def getConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getKnownSubclasses(id: ID): Seq[ID] = ???
+
+    override def getShadowedConversionsFor(id: ID): Seq[ImplicitConversions] = ???
+
+    override def getLinearSupertypesBy(id: ID): Seq[ID] = ???
+
+    override def getCompanionFor(id: ID): ID = ???
+
+    override def getDocElementBy(id: ID): DocElement = docElements.find((x) => x.id == id).get
```

Comment:
Also, there should be package indices, if you are to support the current output model.